sql_generators/glean_usage/templates/baseline_clients_last_seen.view.sql

```diff
@@ -6,7 +6,35 @@ AS
 SELECT
   {% for ut in usage_types %}
     `moz-fx-data-shared-prod`.udf.pos_of_trailing_set_bit(days_{{ ut }}_bits) AS days_since_{{ ut }},
-  {% endfor %}
+  {% endfor %},
+  -- Metrics based on activity
```

Comment:
We have the logic of activity segments in the [clients_last_seen](https://github.com/mozilla/bigquery-etl/blob/15c4835f5a0e2b7d0953f612ff0f2230df789897/sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v1/view.sql#L84-L101) view for Legacy telemetry.
Could we please add it here for Glean telemetry to avoid having it hard-coded in several places and given that these are source of truth views?
```suggestion
  -- Metrics based on activity
  CASE
      WHEN BIT_COUNT(days_active_bits)
        BETWEEN 1
        AND 6
        THEN 'infrequent_user'
      WHEN BIT_COUNT(days_active_bits)
        BETWEEN 7
        AND 13
        THEN 'casual_user'
      WHEN BIT_COUNT(days_active_bits)
        BETWEEN 14
        AND 20
        THEN 'regular_user'
      WHEN BIT_COUNT(days_active_bits) >= 21
        THEN 'core_user'
      ELSE 'other'
    END AS activity_segment,
```